AniRPG.Domain/Entities/Location.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using AniRPG.Domain.Common;
+
+namespace AniRPG.Domain.Entities
+{
+    public class Location : IEntity
+    {
+        public int Id { get; set; }
+        public string Name { get; set; }
+        public string Description { get; set; }
+        public ICollection<Location> Transitions { get; private set;}
```

Comment:
This collection must be initialized.
You should create constructor with code like:
`Transitions = new HashSet<Location>();`

Reply:
thanks, fixed.